api/sheets/get_name.md

```diff
@@ -6,40 +6,36 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the name of a node.
+# Get a node's name
 
-## `=DCGETNAME(dcids)`
+Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
 
-Given a cell or a column range of cells with DCIDs, return the names of those dcids.
+## General information about this method
 
-**Arguments**
-*    `dcids` - DCIDs to get the names of
+**Signature**: `=DCGETNAME(dcids)`
 
-**Returns**
+**Required arguments**:
 
-The names of the DCIDs
+* `dcids` - A list of nodes to query, identified by their DCIDs.
 
 ## Examples
 
-In this example, we find the names of the Data Commons nodes for [California](https://datacommons.org/browser/geoId/06) and [Alameda County](https://datacommons.org/browser/geoId/06001). The function call in the image uses a single call to get the names of "geoId/06" and "geoId/06001", but it is equivalent to typing:
+Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
 
-```
-=DCGETNAME("geoId/06")
-```
+### Example 1: Retrieve the common name of the country of Côte d'Ivoire.
```

Comment:
Maybe leave "Cote d'Ivoire" out from the header here, so as not to confuse between this and "common" name
```suggestion
### Example 1: Retrieve the name of a country by it's DCID.
```

Reply:
This is a great idea! I've incorporated it.